cluster/gce/config-default.sh

```diff
@@ -160,3 +160,6 @@ E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
 
 # Evict pods whenever compute resource availability on the nodes gets below a threshold.
 EVICTION_HARD="${EVICTION_HARD:-memory.available<100Mi,nodefs.available<10%}"
+
+# Optional: custom scheduling algorithm
+SCHEDULING_ALGORITHM_PROVIDER="${KUBE_SCHEDULING_ALGORITHM_PROVIDER:-}"
```

Comment:
nit: why "KUBE_SCHEDULING_ALGORITHM_PROVIDER"?

Let's not introduce yet another variable, and do:
SCHEDULING_ALGORITHM_PROVIDER="${SCHEDULING_ALGORITHM_PROVIDER:-}"